pincer/objects/guild/guild.py

```diff
@@ -602,18 +602,14 @@ async def create_role(
         """
         ...
 
-    async def create_role(
-        self,
-        reason: Optional[str] = None,
-        **kwargs
-    ) -> Role:
+    async def create_role(self, reason: Optional[str] = None, **kwargs) -> Role:
```

Comment:
Makes it go over the limit no?

Reply:
1 to much xd

Reply:
Black changed this for me. I think we should keep it
[See commit](https://github.com/Endercheif/Pincer/commit/4588fad306ed143b0162c4b321cd086ee96b4fb9#diff-13749c83c6f0b5ededf5ce8b663c3aaf98422e0b6330bab883dd54d08dcb98e4)

Reply:
Fair enough